README.md

```diff
@@ -81,6 +85,8 @@ The software is released under the 2-clause BSD license. See separate
 
 
 [Configuration documentation]: docs/Configuration.md
-[upgrade-to-1.0.3]: docs/upgrade-to-1.0.3.md
-[upgrade-from-1.0.x-to-1.1.x]: docs/upgrade-from-1.0.x-to-1.1.x.md
-[upgrade-from-4.0.x-to-5.0.x]: docs/upgrade-from-4.0.x-to-5.0.x.md
+[installation instructions]: docs/Installation.md
```

Comment:
Use a capital "I"? Link labels are case insensitive.

Reply:
”[Installation instructions]" instead of "[installation instructions]"? -- What does it add? In the normal case (when you read the page as Github render it) you will not see that line anyway.

Reply:
The Markdown looks prettier if the casing is consistent in the references. I only pointed it out because I thought you would appreciate it.

Reply:
I have updated. Please re-review.